docs/source/user/core-structures.rst

```diff
@@ -1,28 +1,98 @@
+.. _core-structures:
+
 Core Structures
 ===============
 
-.. contents:: |toctitle|
-    :local:
-
-Overview
---------
-
-OpenDP is focused on creating computations with specific privacy characteristics. These computations are modeled with two core structures in OpenDP: Transformations and Measurements. These structures are in all OpenDP programs, regardless of the underlying algorithm or definition of privacy. By modeling computations in this abstract way, we're able to combine them in flexible arrangements and reason about the resulting programs.
+OpenDP is focused on creating computations with specific privacy characteristics.
+These computations are modeled with two core structures in OpenDP: Transformations and Measurements.
+These structures are in all OpenDP programs, regardless of the underlying algorithm or definition of privacy.
+By modeling computations in this abstract way, we're able to combine them in flexible arrangements and reason about the resulting programs.
 
 Measurement
 -----------
 
 A Measurement is a randomized mapping from datasets to outputs of an arbitrary type.
+Each Measurement consists of a privacy relation and a function.
+
+The privacy relation is used to check if the function is `d_out`-DP on `d_in`-close inputs.
+The privacy relation is qualified by an input metric and an output measure, which give units to the distances `d_in` and `d_out`.
```

Comment:
Also, where we're referencing concepts we haven't encountered yet (especially for domains and relations which don't appear until the next doc), it'd be good to link to the section that covers them.